verticapy/_config/config.py

```diff
@@ -148,6 +153,182 @@ def set_option(key: str, value: Any = None) -> None:
         using iterative functions.
     value: object, optional
         New value of the option.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verticapy.datasets import load_titanic
+        from verticapy import set_option
+
+        titanic = load_titanic()
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import load_titanic
+        from verticapy import set_option
+
+        titanic = load_titanic()
+        html_file = open("figures/_config_config_set_option_1.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_1.html
+
+    .. code-block:: python
+
+        # Computes and displays the total number of elements
+        set_option("count_on", True)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("count_on", True)
+        html_file = open("figures/_config_config_set_option_2.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_2.html
+
+    .. code-block:: python
+
+        # Turns off the footer
+        set_option("footer_on", False)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("footer_on", False)
+        html_file = open("figures/_config_config_set_option_3.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_3.html
+
+    .. code-block:: python
+
+        # Sets the maximum number of columns to display
```

Comment:
Example:
This code block should be two lines and two blocks. You can even add a title.

Customizing the Number of Displayed Columns
----------------------------------------------- (not sure how to do small title like h3)

Let's change the maximum number of elements to display.

...

Let's display the dataset.

...

.. note:: By changing this parameter, we retrieve fewer elements from the database, resulting in faster visualization.